src/ray/core_worker/core_worker.cc

```diff
@@ -126,8 +126,129 @@ std::optional<ObjectLocation> TryGetLocalObjectLocation(
 
 }  // namespace
 
-CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_id)
-    : options_(options),
+TaskCounter::TaskCounter() {
```

Comment:
No-op change for `TaskCounter`, just move the implementation from header file to impl file.